.gitignore

```diff
@@ -4,4 +4,7 @@ logs
 dist
 artifact
 vendor
+languages/*.pot
+**/build/*
+cypress/fixtures
```

Comment:
The fixtures folder is used to add mock data, or standard data for some requests, as can be checked [here](https://docs.cypress.io/api/commands/fixture#Arguments). We also use it on the [Neve repo](https://github.com/Codeinwp/neve/tree/master/e2e-tests/cypress/fixtures), do you think that this can be removed from .gitignore even though we are not using it yet here?

Reply:
@gutoslv I know that is being used for mock data but we don't use any mock data. I thought that since we are not currently using it it can be excluded since it is not part of the project as it is now. But I have removed it now from `.gitignore` if you think that is required. Let me know what you think.

Thank you!

Reply:
I think that even though we are not using the fixtures folder, it's good to keep it outside the .gitignore file since we can probably use it later and forget that it's on the .gitignore file. Thanks for removing it from the .gitignore file!